lib/api.rb

```diff
@@ -0,0 +1,20 @@
+module Api
```

Comment:
for comparison,  the client's base namespace is ManageIQ::API::Client as developed in https://github.com/ManageIQ/manageiq-api-client

maybe we can go with ManageIQ::API::Server ?